src/components/pacbio/PacbioPoolLibraryEdit.vue

```diff
@@ -110,9 +116,20 @@ export default {
       type: Boolean,
       default: false,
     },
+    validated: {
+      type: Boolean,
+      default: false,
+    },
+    notify: {
+      type: Function,
+      required: true,
+    },
```

Comment:
Probably better to add comments on 'validated' field where it is set, how it is used and also on what notify function does

Reply:
Yes agreed, done